openadapt/app/tray.py

```diff
@@ -107,9 +117,13 @@ class SystemTrayIcon:
 
     # storing actions is required to prevent garbage collection
     recording_actions = {"visualize": [], "replay": []}
+    download_complete = Signal(str)
+    download_start_toast = Signal(str)
+    unzipping_started_toast = Signal(str)
```

Comment:
What do you think about adding the suffix `_signal` to each of these?

Reply:
Done